lib/engines/content_block_manager/app/models/content_block_manager/content_block/document/scopes/searchable_by_title.rb

```diff
@@ -4,8 +4,13 @@ module ContentBlock::Document::Scopes::SearchableByTitle
 
     included do
       scope :with_title,
-            lambda { |*keywords|
-              pattern = "(#{keywords.map { |k| Regexp.escape(k) }.join('|')})"
+            lambda { |keywords|
+              split_keywords = keywords.split
+              pattern = ""
+              split_keywords.map do |k|
```

Comment:
not sure if this is better user experience or not! but it's how i would expect a keyword search to work elsewhere.
also might be a neater way of writing but wanted it to be readable

Reply:
Yeah, I think this is good - makes sense to be more flexible